wisdem/floatingse/floating_frame.py

```diff
@@ -632,11 +662,51 @@ def compute(self, inputs, outputs):
             m_sys * outputs["system_structural_center_of_mass"] + m_variable * cg_variable
         ) / (m_sys + m_variable)
 
-        # Transition piece properties
-        m_trans = float(inputs["transition_piece_mass"])
-        r_trans = inputs["platform_Rnode"][itrans_platform]
-        I_trans = m_trans * r_trans ** 2.0 * np.r_[0.5, 0.5, 1.0, np.zeros(3)]
-        outputs["transition_piece_I"] = I_trans
+        # Compute the total cg for the platform and the variable ballast together using a weighted sum approach
+        cg_plat_total = (m_variable * cg_variable + m_platform * cg_platform) / (m_variable + m_platform)
```

Comment:
@gbarter , additionally should the fixed ballast mass be taken into account for cg_plat_total calculations?

Reply:
I think this is already included in the member:I_total value that gets aggregated first:
https://github.com/WISDEM/WISDEM/blob/dakota-inverse/wisdem/floatingse/member.py#L1349